cli/cli.ts

```diff
@@ -1252,6 +1253,31 @@ function buildFolderAsync(p: string, optional?: boolean, outputName?: string): P
     })
 }
 
+function buildCommonSim() {
+    if (!fs.existsSync("node_modules/typescript")) {
+        U.userError("Oops, typescript does not seem to be installed, did you run 'npm install'?");
+    }
+
+    const p = "node_modules/pxt-common-packages";
+    if (fs.existsSync(p)) {
+        pxt.log(`building common-sim...`)
+        dirsToWatch.push(p)
+        return nodeutil.spawnAsync({
+            cmd: "node",
+            args: ["../typescript/bin/tsc"],
+            cwd: p
+        })
+        .then(() => {
+            if (fs.existsSync("built")) {
+                nodeutil.cpR(path.join(p, "built"), "built");
```

Comment:
we should only be copying 2 files: ``pxt-common-sim.d.ts`` and ``pxt-common-sim.js``